go-controller/pkg/ovn/egressip.go

```diff
@@ -1945,6 +1977,27 @@ func (e *egressIPZoneController) deleteEgressIPStatusSetup(name string, status e
 	return podIPs, nil
 }
 
+func (e *egressIPZoneController) getNextHopIP(name string, status egressipv1.EgressIPStatusItem) (string, error) {
```

Comment:
Not a fan of this func name - its not clear what it does differently from reading the fn name versus `getNextHop`.  Maybe attemptToGetNextHopIP or something and add  func doc stating the conditions it returns the IP and when it doesnt, when it returns error and doesnt..

Reply:
ok, changed the method signature addressing your concerns.